app/src/main/java/com/github/polybooks/core/database/implementation/DummySalesQuery.kt

```diff
@@ -39,15 +40,18 @@ class DummySalesQuery(private val sale: List<Sale> = default_sale) : SaleQuery {
 
 
     override fun onlyIncludeInterests(interests: Collection<Interest>): SaleQuery {
-        TODO("Not yet implemented")
+//        TODO("Not yet implemented")
```

Comment:
Same in this file. Sure it might not be implemented in the real version. But in this file, which in my understanding will only be used for testing, the implementation is final. So we should not have all those commented todo's

Reply:
done